src/idb.filesystem.js

```diff
@@ -34,16 +34,17 @@ if (exports.requestFileSystem || exports.webkitRequestFileSystem) {
 }
 
 // Bomb out if no indexedDB available
-const indexedDB = exports.indexedDB || exports.mozIndexedDB ||
+var indexedDB = exports.indexedDB || exports.mozIndexedDB ||
                   exports.msIndexedDB;
 if (!indexedDB) {
   return;
 }
 
-let IDB_SUPPORTS_BLOB = true;
+var IDB_SUPPORTS_BLOB = false;
```

Comment:
But isn't the idea of this work to make filesystem API accessible for (older) browsers that do not support filesystem API? Chrome has it and firefox starts to support it. Its Safari that profits most. The advantages available for led and const are not that big to make a file incompatible with the purpose it should surf. I understand the pragmatism to run with the latest standards, but I kind of disagree with the choice in this case.